policyengine_us/parameters/gov/aca/age_curves/ma.yaml

```
@@ -1,10 +1,9 @@
-description: Age curve factors for ACA premium pricing in Massachusetts, normalized to age 0.
-metadata:
+description: Massachusetts multiplies base marketplace plan premiums by these age-specific factors to determine age-specific premiums.
   type: single_amount
```

bring this back for all (causing the test failure)
```suggestion
metadata:
    type: single_amount
```